src/bolos/cx_hmac.h

```diff
@@ -5,6 +5,9 @@ int cx_hmac_init(cx_hmac_ctx *ctx, cx_md_t hash_id, const uint8_t *key,
 int cx_hmac_update(cx_hmac_ctx *ctx, const uint8_t *data, size_t data_len);
 int cx_hmac_final(cx_hmac_ctx *ctx, uint8_t *out, size_t *out_len);
 
+int cx_hmac_sha512(const unsigned char *key, unsigned int key_len,
```

Comment:
Please remove `cx_hmac_sha512` from `cx.h` (and make appropriate changes such as including `cx_hmac.h` when necessary) in order to prevent this prototype from being declared twice.

Reply:
Looking more closely at the content of files `cx.h`, `cx_hmac.h` and `cx_hash.h`, I can see that there are HMAC-related definitions in all 3 of them. So shouldn't the best move be:
1. moving all the HMAC-related type definitions and function declarations from `cx.h` to `cx_hmac.h`
2. moving the #defines for `sys_cx_hmac_sha256` and `sys_cx_hmac_sha512` from `cx_hash.h` to `cx_hmac.h`
3. Adjusting the #include directives in the source files as necessary.

WDYT?

Reply:
LGTM 👍

Reply:
Done